Flat/interface/PandaAnalyzer.h

```diff
@@ -259,6 +260,7 @@ public :
     Analysis *analysis = 0; //!< configure what to run
     TimeReporter *tr = 0; //!< profile time usage
     float FATJETMATCHDR2 = 2.25;
+    bool applyMCTriggers = false;
```

Comment:
I meant `applyMCTriggers` should be a member of the `Analysis` object, not the `PandaAnalyzer` object. All analysis configuration is done at that level. It is defined in `PandaAnalysis/Flat/interface/AnalyzerUtilities.h`.